doc/docs/2.3.x/how-tos/jupyterlab-extension/index.md

```diff
@@ -1,173 +1,115 @@
-
-    !!! Note 
-
-         Find the latest available tag of the image [`pachyderm/notebooks-user` in DockerHub](https://hub.docker.com/r/pachyderm/notebooks-user/tags){target=_blank} to get the latest copy of the extension.
- 
-         Our image comes with a pre-installed suite of packages, including:
+- Run and test out your pipeline code before creating a Docker image. 
 
-         - The extension `jupyterlab-pachyderm`. 
-         - A custom version of our Command-Line Tool `pachctl`. 
-         - FUSE 
-         - Additionally, it pre-creates the `/pfs` directory files that will be mounted to and grants ownership to the JupyterLab USER.
-        
-         The image is based on a GPU-enabled version of [jupyter/base-notebook¶](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html#jupyter-base-notebook).
+## Before You Start 
 
-- Add the extension to your existing Jupyterlab Docker image:
-
-    !!! Note 
-        Replace the following `${PACHCTL_VERSION}` with the version of `pachctl` that matches your cluster's, and update `<version>` with the release number of the extension.
+- You must have a Pachyderm cluster running.
+- You must install the Jupyterlab Helm chart
+    ```sh
+    helm repo add jupyterhub https://jupyterhub.github.io/helm-chart/
+    helm repo update
+    ```
+--- 
 
-    ```shell 
-    # This runs the following section as root; if adding to an existing Dockerfile, set the user back to whatever you need. 
-    USER root
+## Install The Extension 
 
-    # This is the directory files will be mounted to, mirroring how pipelines are run. 
-    RUN mkdir -p /pfs 
+You can choose between Pachyderm's pre-built image (a GPU-enabled version of [`jupyter/base-notebook`](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html#jupyter-base-notebook)) or add the extension to your own image. Pachyderm's image includes:
 
-    # If you are not using "jovyan" as your notebook user, replace the user here. 
-    RUN chown $NB_USER /pfs
+- The extension jupyterlab-pachyderm.
+- FUSE
+- A pre-created `/pfs` directory that mounts to and grants ownership to the JupyterLab User
+- A `mount-server` binary. 
 
-    # Fuse is a requirement for the mount extension 
-    RUN apt-get clean && RUN apt-get update && apt-get -y install curl fuse 
+### Local Installation (Using Provided Image)
 
-    # Install Pachctl - Set the version of Pachctl that matches your cluster deployment. 
-    RUN curl -f -o pachctl.deb -L https://github.com/pachyderm/pachyderm/releases/download/v${PACHCTL_VERSION}/pachctl_${PACHCTL_VERSION}_amd64.deb 
-    RUN dpkg -i pachctl.deb
+1. Open your terminal.
+2. Run the following:
+    ```sh
+    docker run -it -p 8888:8888 -e GRANT_SUDO=yes --user root --device /dev/fuse --privileged --entrypoint /opt/conda/bin/jupyter pachyderm/notebooks-user:{{ config.jupyterlab_extension_image_tag }}  lab --allow-root
```

Comment:
@chainlink Is it better to ask users to run as `SYS_ADMIN`?

Reply:
We could try `--cap-add=SYS_ADMIN` but i think we still need `--privileged`